reference/docfx.json

```diff
@@ -162,6 +162,7 @@
                 "**/PSWorkflow/**.yml": "SteveL-MSFT",
                 "**/PSWorkflow/**.md": "SteveL-MSFT",
                 "**/PSWorkflowUtility/**.yml": "SteveL-MSFT",
+                "**/ThreadJob/**.md": "PaulHigin",
```

Comment:
Need to replace the `.md` with `.yml`, thus 
```
"**/ThreadJob/**.yml": "PaulHigin"
```

Reply:
fixed